meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb

```diff
@@ -0,0 +1,30 @@
+require mender-connect.inc
+
+RDEPENDS_${PN} = "glib-2.0 mender-client (>= 2.5)"
+
+################################################################################
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender-connect.git;protocol=https;branch=2.0.x"
+
+# Tag: 2.0.0
+SRCREV = "377062297285d84908285bcddab8aaeaa8e4d41c"
+
+# Enable this in Betas, and in branches that cannot carry this major version as
+# default.
+# Downprioritize this recipe in version selections.
+#DEFAULT_PREFERENCE = "-1"
```

Comment:
This is a major version increase so we need to down prioritize it in `dunfell` (obs! not in `master`) same way we are doing for `mender-client` > 3.0